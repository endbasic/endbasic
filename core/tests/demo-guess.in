100
3
10
90
50
yes
50
75
5
yes
50
82
no
