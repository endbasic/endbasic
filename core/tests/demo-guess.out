IncompleteConsole::color(Some(15), Some(4))
IncompleteConsole::clear()

IncompleteConsole::color(Some(14), Some(4))
 Guess the number!
===================
IncompleteConsole::color(Some(15), Some(4))


Alright! I have a secret number between 0 and 100

You have 3 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 10 is too low!
IncompleteConsole::color(Some(15), Some(4))

You have 2 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 90 is too high!
IncompleteConsole::color(Some(15), Some(4))

You have 1 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 50 is too low!
IncompleteConsole::color(Some(15), Some(4))
IncompleteConsole::color(Some(9), Some(4))
Sorry. You lost :-( The secret number was 70
IncompleteConsole::color(Some(15), Some(4))


Alright! I have a secret number between 0 and 100

You have 3 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 50 is too high!
IncompleteConsole::color(Some(15), Some(4))

You have 2 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 75 is too high!
IncompleteConsole::color(Some(15), Some(4))

You have 1 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 5 is too low!
IncompleteConsole::color(Some(15), Some(4))
IncompleteConsole::color(Some(9), Some(4))
Sorry. You lost :-( The secret number was 22
IncompleteConsole::color(Some(15), Some(4))


Alright! I have a secret number between 0 and 100

You have 3 attempts left to guess my number
IncompleteConsole::color(Some(9), Some(4))
Wrong. 50 is too low!
IncompleteConsole::color(Some(15), Some(4))

You have 2 attempts left to guess my number
IncompleteConsole::color(Some(10), Some(4))
Correct. You win! :-)
IncompleteConsole::color(Some(15), Some(4))

IncompleteConsole::color(None, None)
IncompleteConsole::clear()
Score: 1 wins and 2 losses

Thanks for playing

